### PR TITLE
v1.2.1 (#12)

### PR DESCRIPTION
**Pull request by Conjuro**
* v1.2.1 of Streamlink Twitch GUI

* Added a version variable for fewer edits on version upgrades.

* Updated link to changelog

### CODE REVIEW

File: Streamlink Twitch GUI/streamlink-twitch-gui.nuspec [hunk 1]
```diff
@@ -39,6 +39,6 @@ Example: `choco uninstall streamlink-twitch-gui -y -params '"/Purge"'`
     </dependencies>
     <releaseNotes>- Fixed Livestreamer standalone validation. #386
```
Comment (scowalt) [inline Streamlink Twitch GUI/streamlink-twitch-gui.nuspec]: We should remove this part of the release notes.

**Review by scowalt — CHANGES_REQUESTED** [review]
Thanks for taking the time to do this! If you could make the requested change, run choco pack and send me the resulting package file (can you upload the file to a comment on this PR?), I can kick off chocolatey's moderation process.

**Comment by Conjuro** [comment]
Removed the old release notes. Let me know if you want me to squash the commits.

[.nupkg file attached](https://github.com/streamlink/streamlink-twitch-gui-chocolatey/files/900723/streamlink-twitch-gui.1.2.1.zip). Had to rename it to .zip to get GitHub to accept it, just rename it back and it should be good to go.

**Review by scowalt — APPROVED** [review]

**Comment by scowalt** [comment]
Thanks! I'll merge this in when the package is approved, and I'll let you know if there's any feedback we need to iterate on.

https://chocolatey.org/packages/streamlink-twitch-gui/1.2.1

**Pull request merged**